lib/hsa/kernels/rfi_chime_input_sum.cl

```diff
@@ -55,16 +66,35 @@ rfi_chime_inputsum(
         uint address = gy + gz * gy_size;
         if (n * N == 0){
             output[address] = -1.0;
+            output_var[address] = -1.0;
             output_mask[address] = 1;
         } else {
             float new_sq_power_across_input = sq_power_across_input[0] * cf * cf;
             float SK = ((n + 1) / (n - 1)) * ((new_sq_power_across_input / (n * N)) - 1);
+
+            // Calculate the truncation bias correction for the SK value
+            const float var = var_across_input[0] * one_over_256;
+            const float var2 = var * var;
+            const float var_sqrt = sqrt((double)var);
```

Comment:
Arash might be able to provide you a polynomial fit against var instead of the rms, which would save the sqrt.

Reply:
Arash is trying to do a fit for this one now. I think he's going to put the results in chime-experiment/RTPipeline#73